spec/support/matchers/have_column.rb

```diff
@@ -8,7 +8,7 @@
     column && column.default.to_s == @default.to_s
   end
 
-  failure_message_for_should do |columns|
+  failure_message do |columns|
```

Comment:
This is also in #1727, and i have the same question: Is this backwards compatible with RSpec 2? If not, can you remove this and reference it separately, as it's out of scope for this PR?